### PR TITLE
Update CP1.md

### PR DESCRIPTION
**Pull request by Rhanok09**
- templates for the RFCs added
- edited some formatting
- added hyperlinks to the Forum, OpenSquare and the Proposal Repository

### CODE REVIEW

File: cps/CP1/CP1.md [hunk 1]
```diff
@@ -22,9 +22,54 @@ The next step is the funding process, which will be an on-chain Treasury Proposa
 | --- | --- | :---: |
 |5|Create an on-chain Treasury proposal|-|
 
+### TEMPLATE
+```
```
Comment (Rhanok09) [inline cps/CP1/CP1.md:26]: I wanted to frame the template in a way but couldn't figure out how to do it so I used ``` ... ```

File: cps/CP1/CP1.md [hunk 2]
```diff
@@ -22,9 +22,54 @@ The next step is the funding process, which will be an on-chain Treasury Proposa
...
+Title: *RFC: [TITLE OF PROPOSAL]*
+
+Tags: *cp-1, month, year*
+```
+#### Body
```
Comment (ImdioR) [inline cps/CP1/CP1.md:31]: i think that will be better to write

#### Body of the RFC must contain: 

What do you think?

Comment (Rhanok09) [reply cps/CP1/CP1.md:31]: Yes, I will add it.

**Review by ImdioR — APPROVED** [review]
Reviewed

**Pull request merged**